### PR TITLE
CRM7 Caseworker: Change defendant details design

### PR DESCRIPTION
**Pull request by ivanELEC**
## Description of change

[Link to relevant ticket](https://dsdmoj.atlassian.net/browse/CRM457-1986)

## Notes for reviewer
[Update to e2e tests here](https://github.com/ministryofjustice/nsm-e2e-test/pull/152)

### CODE REVIEW

File: app/view_models/nsm/v1/defendant_details.rb [hunk 1]
```diff
@@ -29,45 +29,51 @@ def main_defendant_maat
         main_defendant['maat']
       end
 
+      def main_defendant_value
+        if main_defendant_maat.present?
```
Comment (patrick-laa) [inline app/view_models/nsm/v1/defendant_details.rb:33]: You could probably DRY this up by doing `construct_value(defendants.find { _1['main'] })`

File: app/view_models/nsm/v1/defendant_details.rb [hunk 2]
```diff
@@ -29,45 +29,51 @@ def main_defendant_maat
...
 
       def rows
         { title:, data: }
       end
+
+      private
+
+      def construct_value(defendant)
+        if defendant['maat'].present?
+          multiline_text("#{construct_name(defendant)}\n#{defendant['maat']}")
```
Comment (patrick-laa) [inline app/view_models/nsm/v1/defendant_details.rb:72]: It feels a little bit heavy-handed to modify the string twice - first to add a newline and then to convert the newline into a `<br>`. You could do it in one step with `safe_join([construct_name(defendant), tag.br, defendant['maat']])`

File: app/view_models/nsm/v1/defendant_details.rb (hunk 1 above)
Comment (ivanELEC) [reply app/view_models/nsm/v1/defendant_details.rb]: updated

File: app/view_models/nsm/v1/defendant_details.rb (hunk 2 above)
Comment (ivanELEC) [reply app/view_models/nsm/v1/defendant_details.rb]: updated

**Review by patrick-laa — APPROVED** [review]

**Pull request merged**